pom.xml

```diff
@@ -31,6 +31,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>3.1.1</version>
```

Comment:
you don't need this, take a look at 15 row

Reply:
I'll send a screencsot to Slack to explain the reason of this

Reply:
remove this line, since we figured out it works without it

Reply:
done